str/trtools/stat_str_runner.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+# pylint: disable=duplicate-code
+"""
+This script runs statSTR() from TRTools package on a single/merged STR vcf file and outputs various statistics
+
+For example: 
+analysis-runner --access-level test --dataset tob-wgs --description 'tester' --output-dir 'tester' statSTR_runner.py --caller=eh --file-path=gs://cpg-tob-wgs-test/hoptan-str/mergeSTR/mergeSTR_2_samples_gangstr.vcf
+
+Required packages: sample-metadata, hail, click, os
```

Comment:
I think each of your scripts has this in comments - it's better to formalise this by having a requirements.txt file in the project root, ideally with versions of the relevant libraries.

sample-metadata>=X.Y.z
hail>=0.2.105
...

they can then be installed in bulk with

```
pip install -r requirements.txt
```